pyserini/server/mcp/tools.py

```diff
@@ -98,4 +98,68 @@ def get_index_status(index_name: str) -> dict[str, Any]:
         Returns:
             Dictionary with index information.
         """
-        return controller.get_status(index_name)  
\ No newline at end of file
+        return controller.get_status(index_name)  
+    
+    @mcp.tool(
+        name="fuse_search_results",
+        description="Performs normalization fusion on search results to improve ranking."
+    )
+    def fuse_search_results(
+        hits1: list[DenseSearchResult], 
```

Comment:
Why `DenseSearchResult`

Reply:
Strange, @lintool naming issue we should make an issue for and eventually return to. I will resolve for speed rn.